### ISSUE TITLE
After upgrading to 2.4.12: Quote sign not escaped in attributes

### ISSUE THREAD

**Issue by groe** [ohler55/ox#170]
After upgrading ox 2.4.11 to 2.4.12, one of my tests fails:

```ruby
b=Ox::Builder.new
b.element(:a, title: '" href="http://evil.example.org')
b.pop
b.to_s == "<a title=\"&quot; href=&quot;http://evil.example.org\"/>\n"
# => false
b.to_s
# => "<a title=\"\" href=\"http://evil.example.org\"/>\n"
```

**Comment by groe** [comment]
(This is also present in 2.4.13b1 from github btw, but 2.4.12 was the first version it appeared)

**Comment by groe** [comment]
Fixed in ` 2.4.13b2`, thanks @ohler55!
